src/main/java/io/lakefs/routerfs/RouterFileSystem.java

```diff
@@ -252,4 +300,15 @@ private Path createSchemedPath(Path p) {
         }
         return p;
     }
+
+    private static FileStatus switchFileStatusPathPrefix(FileStatus fileStatus, String fromPrefix, String toPrefix) {
```

Comment:
Can you please document the reasoning for this method? It's an iSight worth documenting